src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationServiceImpl registration = new RegistrationServiceImpl();
+    private StorageDaoImpl storage;
+    private User alice = new User();
+    private User bob = new User();
+    private User jack = new User();
+
+    @Test
+    void addNullUser_NotOk() {
+        assertThrows(RuntimeException.class, () -> {
+            registration.register(null);
```

Comment:
lets add user null check in register method. (getting NPE it's not valid case). Also lets add checks for all user fields for null, so we wont get NPE